src/_includes/alert.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-light {{ include.display }}" role="alert">
+  <div class="container fs-7">
+    DGS has issued a Draft RFP for GTFS-Realtime software so public transit agencies can provide riders with real-time fleet
+    location. Responses due by 10/12. <a href="https://caleprocure.ca.gov/event/77601/0000028520" target="_blank">Learn more</a>.
```

Comment:
@machikoyasuda I noticed that the link in @mrose914 's https://github.com/cal-itp/mobility-marketplace/issues/472#issuecomment-1739977023 contained a part with `https://urldefense.com...`. I'm not sure what it's for, but I'm wondering if it was intentional to have that specific URL

Reply:
@angela-tran I'm guessing that URL Defense link thing came from a CA State employee emailing this link to Marissa.

I don't think we need it b/c the Cal-ITP LinkedIn announcement about this also just links directly to https://caleprocure.ca.gov/event/77601/0000028520